drivers/spi/Kconfig.mcux_lpspi

```diff
@@ -19,4 +19,18 @@ config SPI_MCUX_LPSPI_DMA
 	help
 	  Enable the SPI DMA mode for SPI instances
 	  that enable dma channels in their device tree node.
+
+if SPI_RTIO
```

Comment:
you could have a "depends on SPI_RTIO instead of such if/endif.

Reply:
the if/endif is more scalable syntax

Reply:
Can you help me understand the advantage of "depends on" over "if/endif"?
Perhaps I missed something here.